.github/workflows/haskell.yml

```diff
@@ -0,0 +1,53 @@
+name: Haskell
+on:
+  push:
+  pull_request:
+
+jobs:
+  ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        ghc: [8.10.7]
+    runs-on: ubuntu-latest
+    container:
+      # ubuntu with ghcup already installed
+      image: ghcr.io/facebookincubator/hsthrift/ci-base:ghcup
+      options: --cpus 2 --security-opt=seccomp=unconfined
```

Comment:
Hmm, the commit message has some details:

> hhvm makes a couple of syscalls that are blocked by default on github's docker seccomp policy. We need to allow those. Unblocking all for now (see https://docs.docker.com/engine/security/seccomp/ )

I think we can disable this here since we aren't going to be using HHVM.